src/shivaniLikhia/Assignment28.java

```diff
@@ -0,0 +1,178 @@
+/*Assignment - 28 : 25th Sep'2022
+Program 1 : [10 mins - 13 mins]
+Return count of vowels in a given String using switch case.
+input : TechnOcredits
+output : 4
+Program 2 : [10 mins]
+Return first vowel from given String.
+input : technocredits
+outout : e
+Program 3: [15 mins]
+Print count of each vowels in a given String using switch case.
+input : technocredits
+output : e -> 2
+         o -> 1
+		 i -> 1
+		 
+Program 4: [15 mins]
+Print count of each vowels in a given String Array using switch case.
+input : {"technocredits", "swati", "aashvi"}
+output : vowel count in technocredits:
+		 e -> 2
+         o -> 1
+		 i -> 1
+		 vowel count in swati:
+		 a -> 1
+         i -> 1
+		 vowel count in aashvi:
+		 a -> 2
+         i -> 1
+int count = 0;
+for(int index=0;index<str.length();index++){
+	char ch = str.charAt(index);
+	
+	switch(ch){
+		case 'a':
+		case 'A':
+			count++;
+			break;	
+	}
+}
+
+*/
+
+package shivaniLikhia;
+
+public class Assignment28 {
+
+	int getCountOfVowelInString(String str) {
+		int count = 0;
```

Comment:
Also you can use toLowerCase(), so that you don't have to write capital n small together